packages/core/src/layout/TextAtomicBox.ts

```diff
@@ -8,10 +8,9 @@ export default class TextAtomicBox extends AtomicBox {
   protected height?: number;
   protected content: string = '';
   protected textStyle: TextStyle = {
-    fontFamily: 'sans-serif',
+    fontFamily: 'Source Sans Pro',
```

Comment:
I encountered issues with using system default fonts, where canvas measurements were inconsistent with the DOM. Setting hard-coded font to Source Sans Pro for now, will make this configurable later.